meta-lmp-base/recipes-support/lmp-device-auto-register/lmp-device-auto-register.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "A systemd oneshot helper to auto register a device"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+RDEPENDS_lmp-device-auto-register = "lmp-device-register"
```

Comment:
You can change this to 'RDEPENDS = "lmp-device-register"' instead, no need for _...